lib/scripts/lingui.ts

```diff
@@ -0,0 +1,24 @@
+// Wrapper around lingui command that exits with a non zero value if an error is detected
+import * as child from "child_process";
+
+const args: string[] = process.argv.splice(2);
+const command = `NODE_ENV=test npx lingui ${args.join(" ")}`;
+child.exec(command, (error, stdout, stderr) => {
+  if (error) {
+    // Handle exec command error (lingui command not found?)
+    console.log(`error: ${error}`);
+    throw error;
+  }
+  if (stderr) {
+    // Display command standard error
+    console.log(`stderr: ${stderr}`);
+    // This signals that something wrong happened during the command (BEWARE OF CHANGES IN LINGUI OUTPUT)
+    if (stderr.includes("Cannot process file")) {
+      process.exit(1);
```

Comment:
Here it would be super good if the locale folders would be cleaned again
with e.g. `git checkout site/locale/ && git checkout app/locale` 
to reset all updated changes.

Problem is that lingui updated the files anyways, even if there was an error.

To make sure that this error is not overseen we can at least reset all changed files and the next developer who runs this command notices that the script did nothing and takes a look at the console output.